ecosystem/sep-0009.md

```diff
@@ -70,6 +70,14 @@ Name | Type | Description
 
 ### Organization KYC fields
 
+These field names should always be used as strings. For example:
+```json
+{
+	"organization.name": "Stellar Development Foundation"
+}
+```
+The dot notation (`organization.<field>`) is _not_ an indication that the fields described should be nested under a top-level `"organization"` object.
+
```

Comment:
💡 I think this could be a new section that isn't tied to the organization fields. e.g. `## Sub-fields and dot-notation`